Update tutorial-template-bindings.asciidoc (#877)
vaadin/flow-and-components-documentation: documentation/polymer-templates/tutorial-template-bindings.asciidoc

customElements.define(MyTemplate.is, MyTemplate);
----

== Binding Property Values

> To set an element property value based on a model, use the property name in attribute format, that is use dash-case (not camelCase).

Reviewer: ```suggestion
To set an element property value based on a model, use the property name in the attribute format, by using dash-case `my-property-name` (not camelCase `myPropertyName`).
```
